app/app/components/Button.module.css

```diff
@@ -0,0 +1,41 @@
+/* Button.module.css */
+
+.button {
+    padding: 12px 24px; /* Padding inside the button */
```

Comment:
The comments here can be deleted. This is just simple CSS so no need for clarification here.